core/src/main/java/hivemall/GeneralLearnerBaseUDTF.java

```diff
@@ -220,6 +235,23 @@ protected PrimitiveObjectInspector processFeaturesOI(@Nonnull ObjectInspector ar
         this.featureListOI = (ListObjectInspector) arg;
         ObjectInspector featureRawOI = featureListOI.getListElementObjectInspector();
         HiveUtils.validateFeatureOI(featureRawOI);
+        if (featureRawOI instanceof JavaStringObjectInspector) {
```

Comment:
We need to consider other possibilities for StringObjectInspector.
https://gyazo.com/4c7c3efb22f441db09f614457235b5c0

I'll handle this issue.